syntax = "proto3";
package DNNServer;

service DNNServer { rpc SendQuery(Request) returns (Response); }

message Request {
  int32 id = 1;
  string model = 2;
  int32 bs = 3;
  int32 seq_len = 4;
  int32 qos_target = 5;
}

message Response {
  int32 node_id = 1;
  bool accepted = 2;
}
